client/src/holdings/AddHoldingForm.tsx

```diff
@@ -67,16 +68,18 @@ export function AddHoldingForm(props: AddHoldingFormProps) {
             <Field component={TextField} name='exchange' label='Exchange' />
             <Tooltip
               title='This is the currency the instrument is priced in on FinKi'
-              placement='top'
+              placement='left-end'
               arrow
               interactive>
-              <Field as='select' name='currency'>
-                {CURRENCIES.map((c) => (
-                  <option key={c} value={c}>
-                    {c}
-                  </option>
-                ))}
-              </Field>
+              <span>
```

Comment:
The problem with this is that it squashes the select, so I don't think we want to do that. I had a quick look but couldn't find a good solution so unless you can find one I think just remove the tooltip from this element, it's unusual to have them on selects anyway

Reply:
@willbeaufoy I've deleted both Select Tooltips. Can relook at adding a helper label maybe when we restyle the forms.